FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.HealthCheck/Checkers/AzureContainerRegistryHealthChecker.cs

```diff
@@ -23,22 +21,24 @@ public class AzureContainerRegistryHealthChecker : BaseHealthChecker
         private const string MediatypeV2Manifest = "application/vnd.docker.distribution.manifest.v2+json";
         private readonly string _imageReference;
         private readonly IContainerRegistryTokenProvider _containerRegistryTokenProvider;
+        private readonly string _name;
 
         public AzureContainerRegistryHealthChecker(
-            IOptions<SchemaConfiguration> schemaConfiguration,
+            string prefix,
+            string imageReference,
             IContainerRegistryTokenProvider containerRegistryTokenProvider,
             ILogger<AzureContainerRegistryHealthChecker> logger)
-            : base(HealthCheckTypes.AzureContainerRegistryCanRead, false, logger)
+            : base(prefix + HealthCheckTypes.AzureContainerRegistryCanRead, false, logger)
```

Comment:
not sure 'Prefix' is for?

Reply:
to distinguish ACR checkers for schema or filter.